app/views/content/international-qualified-teacher-status/_after-accordion.md

```diff
@@ -4,12 +4,21 @@ Learn more about [iQTS policy](https://www.gov.uk/government/publications/intern
 
 Learn more about [other routes to qualified teacher status (QTS) and teaching in England](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk#apply-to-the-teaching-regulation-agency-tra).
 
-[Come to England to teach if you're a teacher from outside the UK](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk) gives information and advice to international teachers who’d like to teach in England.
+[Find help and guidance for international teachers who'd like to teach in England.](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk)
 
 If you already live in the UK, you should consider [applying for teacher training leading to QTS in England](/steps-to-become-a-teacher). 
 
-This guidance relates to teaching in England. Different rules apply to teaching in [Wales](https://www.ewc.wales/site/index.php/en/registration/practitioners-trained-outside-wales.html), [Scotland](https://teachinscotland.scot/become-a-teacher/qualified-outside-scotland/) and [Northern Ireland](https://gtcni.org.uk/registration/getting-registered).
+This guidance relates to teaching in England. Different rules apply to teaching in [Wales,](https://www.ewc.wales/site/index.php/en/registration/practitioners-trained-outside-wales.html) [Scotland](https://teachinscotland.scot/become-a-teacher/qualified-outside-scotland/) and [Northern Ireland](https://gtcni.org.uk/registration/getting-registered).
 
 ## Contact
 
-For help applying for iQTS or more information, contact us on iqts.pilot@education.gov.uk
+To get advice about iQTS, you can:
+
+• call 0800 389 2500 for free from anywhere in the world, Monday to Friday between 8:30am and 5:30pm (UK local time)
+
+• use the live chat service at the bottom of this page
+
+• sign up to [get an adviser](https://adviser-getintoteaching.education.gov.uk/) – you’ll be asked for your phone number, email and location so a teacher training adviser can call you back in your time zone
```

Comment:
Ideally this link needs to have slightly different link text eg "get a dedicated adviser" or "get a teacher training adviser" or "get an expert adviser"


Otherwise we will get flagged for accessibility - identical link text going to different links (even if ultimately they are the same destination)